app/lib/appointment_row.rb

```diff
@@ -0,0 +1,54 @@
+class AppointmentRow
+  ROW_INDICES = {
+    guider: 7,
+    date: 10,
+    time: 13,
+    duration: 14,
+    cancellation_date_time: 21,
+    notes: 29,
+    opt_out_of_market_research: 30,
+    status: 34,
+    booking_reference: 40,
+    first_name: 41,
+    last_name: 42,
+    email: 43,
+    mobile: 54,
+    phone: 55,
+    memorable_word: 61
+  }.freeze
+
+  def initialize(row)
```

Comment:
I'm doing that as a pre-requisite when handling the CSV. It does actually require a little clean-up before it's good to import. This needs documenting elsewhere.